include/chaiscript/dispatchkit/dispatchkit.hpp

```diff
@@ -782,7 +782,19 @@ namespace chaiscript
 
         Boxed_Value call_function(const std::string &t_name, const std::vector<Boxed_Value> &params) const
         {
-          return dispatch::dispatch(get_function(t_name), params, m_conversions);
+          try {
+            return dispatch::dispatch(get_function(t_name), params, m_conversions);
+          }
+          catch(chaiscript::exception::dispatch_error&) {
+            auto functions = get_function("method_missing");
+            if (!functions.empty()) {
+              std::vector<Boxed_Value> tmp_params;
+              tmp_params.push_back(var(t_name));
```

Comment:
Can you merge these lines into:

```
std::vector<Boxed_Value> tmp_params{var(t_name)};
```

It's *slightly* more efficient, but also more succinct.